x/stake/types/test_utils.go

```diff
@@ -105,37 +113,36 @@ func AssertInvariants(t *testing.T, msg string,
 		pOrig.UnbondedTokens, pOrig.BondedTokens,
 		pMod.UnbondedTokens, pMod.BondedTokens, tokens)
 
-	// nonnegative bonded shares
+	// Nonnegative bonded shares
```

Comment:
Technically the coding standards say not to capitalize incomplete sentences, but if you do capitalize, make sure to punctuate. This one could go either way ...

Reply:
Interesting -- I haven't read this before. I always found it more...pleasant to read. Golang's source code itself seems to be inconsistent. I'll revert 👍